resources/scss/styles.scss

```diff
@@ -0,0 +1,651 @@
+    margin-bottom: 8px;
+    width: 90vw;
+  }
+
+  .program-icon {
+    float: left;
+    vertical-align: middle;
+    display: inline-block;
+    height: 58px;
+    position: relative;
+  }
+
+  .program-title {
+    float: left;
+    text-align: center;
+    font-size: 14px;
+    width: 26%;
+    padding-left: 7px;
+    line-height: 52px;
+    color: $primary;
+    font-weight: bold;
+    vertical-align: middle;
+    display: inline-block;
+    height: 58px;
+    position: relative;
+  }
+
+  .program-description {
+    width: calc(74% - 6.5em);
+    float: left;
+    font-size: 12px;
+    padding: 12px 0 8px 0;
+    text-align: left;
+    word-break: keep-all;
+    overflow: auto;
+    height: auto;
+    line-height: 1.5em;
+    vertical-align: middle;
+    display: inline-block;
+    position: relative;
+  }
+}
+
+.program-join {
+  text-align: center;
+  color: white;
+  background-color: $primary;
+  padding: 30px;
+}
+
+.stories-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+}
+
+.story {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 20px 0;
+}
+
+.story-img {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 180px;
+  height: 180px;
+}
+
+.story-text {
+  display: flex;
+  flex-direction: column;
+  padding-left: 20px;
+}
+
+.story-title {
+  font-size: 22px;
+  font-weight: bold;
+}
+
+.story-teller {
+  font-size: 14px;
+  font-weight: bold;
+}
+
+.story-hashtag {
+  font-size: 13px;
+  color: $primary;
+  font-weight: bold;
+}
+
+.story-team {
+  font-size: 13px;
+  margin-top: 8px;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.story-members {
+  font-size: 13px;
+  margin-top: 4px;
+}
+
+.about-intro {
+  display: flex;
+  flex-direction: column;
+  background-image: url('../images/about-bg-01.png');
+  background-repeat: no-repeat;
+  height: auto;
+  background-color: $lighter;
+}
+
+.about-welcome {
+  text-align: center;
+}
+
+.about-title {
+  text-align: center;
+}
+
+.about-message {
+  border: $light 4px solid;
+  background-color: white;
+}
+
+.about-contact {
+  font-size: 13px;
+  line-height: 150%;
+  text-align: center;
+}
+
+.about-event-paragraph {
+  font-size: 13px;
+  text-align: justify;
+  margin: 0 auto 16px auto;
+}
+
+.about-prev-group {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.about-prev-img {
+  width: 100%;
+  padding: 10px 20px;
+}
+
+.about-partners {
+  background-color: $dark;
+  color: white;
+}
+
+.about-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.about-item {
+  padding: 8px;
+  font-size: 20px;
+  color: $lighter;
+  list-style: none;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ] In a mobile device, about page, the footer elements should be positioned horizontally, please fix that.
- [ ] Also the background is white, kindly fix that.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/163456645-ea6035bc-93e4-46d2-9356-50321fea6b18.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/163456828-b11a343c-5898-40be-95e2-eb54023305fb.png)